README.md

```diff
@@ -1,2 +1,281 @@
-# Terraform-nomad-presto
+<p align="center">
+ <h2 align="center">Terraform-nomad-presto</h2>
+ <p align="center">Terraform module with example</p>
+ <a href="https://github.com/fredrikhgrelland/vagrant-hashistack/releases">
+   <img alt="Releases" src="https://img.shields.io/badge/dynamic/json?label=with%20vagrant-hashistack&query=%24.current_version.version&url=https%3A%2F%2Fapp.vagrantup.com%2Fapi%2Fv1%2Fbox%2Ffredrikhgrelland%2Fhashistack"/>
+ </a>
+</p>
 
+#
+
+## Contents
+1. [Compatibility](#compatibility)
+2. [Usage](#usage)
```

Comment:
Wondering if we should have the prerequisites section in every tearraform-nomad-module readme that should have a link to https://github.com/fredrikhgrelland/vagrant-hashistack-template#install-prerequisites
Mainly with regarsd to running the 'make install' command as the first command, if the user has not run that on their machine before.

Reply:
Hm, that is a good point. I agree

Reply:
Added